### PR TITLE
Adjust license copyright and contibuting 

### PR DESCRIPTION
**Pull request by i300543**
in accordance with the new DCO and Reuse compliance requirements

https://jam4.sapjam.com/wiki/show/JBQV5da04GwOp45levWo10

### CODE REVIEW

**Review by michawai — COMMENTED** [review]
- You should remove https://github.com/SAP/vscode-logging/edit/dcoCompliance/CONTRIBUTING.md#L36 which mentions CLA
- I recommend to add a compliance job to run on PR and on master.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ A sub-package **may** have an additional CONTRIBUTING.md file if needed.
 
 ## Legal
 
-All contributors must sign the CLA
+All contributors must sign the Developer Certificate of Origin (DCO)
 
 - https://cla-assistant.io/SAP/vscode-logging
```
Comment (michawai) [inline CONTRIBUTING.md:10]: Should be removed/updated

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# root files for https://reuse.software
```
Comment (bd82) [inline .gitignore:1]: The negation pattern must appear after the inclusion pattern.
- I fixed this in my original PRs

File: packages/logger/package.json [hunk 3]
```diff
@@ -6,6 +6,12 @@
     "vscode",
     "logger"
   ],
+  "files": [
+    ".reuse",
+    "LICENSES",
+    "lib",
```
Comment (bd82) [inline packages/logger/package.json:12]: i think `lib` is always included

**Review by bd82 — APPROVED** [review]

**Pull request merged**